ads/zucks.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function zucks(global, data) {
+  try {
+    validateData(data, ['frame_id']);
+    global.context.renderStart(data);
```

Comment:
Internally we are calling http request and do not expose any global event to hook for.
So the example in `fakead3p.js` fit for us.

We can instead use something like `XMLHttpRequest.prototype.open` to hook for all http request but I think it's not neccessary.

Let me know what you think